spotinst/client/client.go

```diff
@@ -79,3 +79,15 @@ func (c *Client) logResponse(resp *http.Response) {
 		}
 	}
 }
+
+// Do runs a request with our client.
+func (c *Client) DoOrg(ctx context.Context, r *Request) (*http.Response, error) {
```

Comment:
can we use this for user creation as well? if yes keep the same name or else please change accordingly

Reply:
yes, we can use it for user level APIs also.